packages/wallets/slope/src/adapter.ts

```diff
@@ -104,7 +105,7 @@ export class SlopeWalletAdapter extends BaseMessageSignerWalletAdapter {
             this._connecting = true;
 
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            const wallet = new window!.Slope!();
+            const wallet = window.slopeApp || new window!.Slope!();
```

Comment:
Got it, I'll revert these changes. However, per https://github.com/solana-labs/wallet-adapter/pull/326#issuecomment-1052747446, I don't see how the polling for `window.slopeApp` is supposed to work with the `connect` function. Won't it error because `window.Slope` is undefined?

Reply:
Ah, I think I understand your above comment now. Still, I don't like the fact that we are relying on `window.slopeApp` to be defined and ultimately using `window.Slope` I guess instead of the non-null assertion I'll just add another check.